recipes/license_finder.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+append_to_file 'Gemfile', after: /gem 'rubocop-infinum'.*\n/ do
+  <<~HEREDOC
+    gem 'license_finder', require: false
+  HEREDOC
+end
+
+append_to_file '.overcommit.yml', after: /command: \['bundle', 'exec', 'rubocop'\].*\n/ do
+  <<~HEREDOC
+
+    LicenseFinder:
+      enabled: true
+      on_warn: fail
+      command: ['bundle', 'exec', 'license_finder']
+      requires_files: true
+      include:
+        - 'Gemfile*'
+        - 'package.json'
```

Comment:
I think this needs to be indented one more level to the left to be correctly nested under `PreCommit` section.